pkg/plugins/generic/generic_plugin.go

```diff
@@ -233,6 +238,46 @@ func isCommandNotFound(err error) bool {
 	return false
 }
 
+// addToDesiredKernelParams Should be called to queue a kernel param to be added to the node.
+func (p *GenericPlugin) addToDesiredKernelParams(kparam string) {
+	if _, ok := p.DesiredKernelParams[kparam]; !ok {
+		glog.Infof("generic-plugin addToDesiredKernelParams(): Adding %s to desired kernel params", kparam)
+		// element "uint" is a counter of number of attempts to set the kernel param
+		p.DesiredKernelParams[kparam] = 0
```

Comment:
why do we need a counter?

Reply:
It is an inexpensive way to debug how many times we have tried to set the kernel arguments without success. I could change it to a bool, but we would lose that debug element.